server/sandbox_stop_linux.go

```diff
@@ -45,6 +45,11 @@ func (s *Server) stopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	stopMutex.Lock()
 	defer stopMutex.Unlock()
 
+	// Clean up sandbox networking and close its network namespace.
+	if err := s.networkStop(ctx, sb); err != nil {
```

Comment:
This is where OCI CNI fails with modifying the `ip6tables` chain. I cannot reproduce it on my local machine but in the CI every container stop does not seem to work any more. I think it might be related to docker networking and we may want to switch the integration tests to run completely without docker :thinking: